README.md

```diff
@@ -46,7 +46,8 @@ Quick start in your local: [quickstart](./docs/guides/quickstart/quickstart-full
 
 Please follow How to install Erda.
 
-- [Installation Document](./docs/guides/deploy/How-to-install-the-Erda.md)
+- [installation document](./docs/guides/deploy/How-to-install-the-Erda.md)
```

Comment:
should not change this line, keep `Installation Document`, not `installation document`